jobs/e2e_node/benchmark-config.yaml

```diff
@@ -159,39 +159,60 @@ images:
     tests:
       - 'resource tracking for 90 pods per node \[Benchmark\]'
 
-  ubuntustable2-resource1:
+  # NOTE: recommended versions of the ubuntu-gke images for GKE 1.14, 1.15 and 1.16 clusters
+  # were taken from https://cloud.google.com/kubernetes-engine/docs/release-notes
+  ubuntustable1-resource1:
     image: ubuntu-gke-1804-1-16-v20200330
     project: ubuntu-os-gke-cloud
     machine: n1-standard-1
     tests:
       - 'resource tracking for 0 pods per node \[Benchmark\]'
-  ubuntustable2-resource2:
+  ubuntustable1-resource2:
     image: ubuntu-gke-1804-1-16-v20200330
     project: ubuntu-os-gke-cloud
     machine: n1-standard-1
     tests:
       - 'resource tracking for 35 pods per node \[Benchmark\]'
-  ubuntustable2-resource3:
+  ubuntustable1-resource3:
     image: ubuntu-gke-1804-1-16-v20200330
     project: ubuntu-os-gke-cloud
     machine: n1-standard-1
     tests:
       - 'resource tracking for 90 pods per node \[Benchmark\]'
 
-  ubuntustable1-resource1:
-    image: ubuntu-gke-1804-1-16-v20200330
+  ubuntustable2-resource1:
+    image: ubuntu-gke-1804-1-15-v2020033
```

Comment:
This seems to be a typo. Should it be "v20200330" here and elsewhere?

Reply:
@Karan thank you for the review. Should bee fixed now.